queries/earth.jinja2

```diff
@@ -21,14 +21,14 @@ FROM
 WHERE
     mz_earth_min_zoom <= {{ zoom }} AND
     {{ bounds|bbox_filter('the_geom') }}
-GROUP BY mz_properties
+GROUP BY gid
```

Comment:
Alas no, postgresql complained about it.

If it's safe to assume that we always have a `kind`, would it be better to group on that?